### PR TITLE
Timeout bash function

### PR DESCRIPTION
**Pull request by sjwoodman**
Added bash function for `timeout` as not natively available on Mac

### CODE REVIEW

File: etc/scripts/install.sh [hunk 1]
```diff
@@ -23,6 +23,8 @@ REPO_DIR=$DIR/.repos
 rm -rf "$REPO_DIR"
 mkdir -p "$REPO_DIR"
 
+function timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
```
Comment (matzew) [inline etc/scripts/install.sh:26]: does that work on Linux ? 

why not instaling timeout via brew ?

Comment (sjwoodman) [reply etc/scripts/install.sh:26]: Yes, it worked on linux - tested on Centos and Ubuntu. Would rather avoid adding homebrew installation as you have to hack the $PATH too

Comment (jcrossley3) [reply etc/scripts/install.sh:26]: as much as i admire the boldness of using perl, it greatly offends my sensibilities :)

Comment (sjwoodman) [reply etc/scripts/install.sh:26]: What would you prefer? :-)

IMO the coreutils package is worse as you either need to edit the script to use gtimeout or add gnubin to the front of the path.

**Comment by jcrossley3** [comment]
I appreciate the contribution, really I do! I just want to limit the number of languages involved ;)

**Pull request closed without merging**